src/components/farmedList.vue

```diff
@@ -87,7 +95,28 @@ export default defineComponent({
   methods: {
     formatDate(date: Date) {
       return formatDistanceToNowStrict(date)
+    },
+    displayRewardAddress() {
+      const addr: string | null = LocalStorage.getItem("rewardAddress")
+      if (addr == null) {
+        const config = appConfig.getAppConfig()
+        if (config) {
+          const { farmerPublicKey } = config.account
+          const addr2: string = addr ?? farmerPublicKey
+          return this.rewardAddress = addr2
+        }
+      return this.rewardAddress = ""
+      } else {
+        return this.rewardAddress = addr
```

Comment:
1) Indentation is odd
2) Don't mix assignment and return, please

Reply:
It wasn't working when I deleted the `return`. Will try to solve it, else I will ask help from you or Serge.

Reply:
Well, you can always split those into 2 lines:
```typescript
this.rewardAddress = addr
return this.rewardAddress
```